docs/infra/Autosubmit-bot.md

```diff
@@ -11,44 +14,54 @@ When a developer opens a pull request against any of the currently supported rep
 4. If all validations are successful the Autosubmit bot will merge the pull request.
 
 ## What Validations does the Autosubmit Bot Perform
+
 ### Code Reviews
+
 The following are the rules for code reviews:
 * If the author is a member of the GitHub `flutter-hackers` org then they will need **at least 1 other review** from a member of the `flutter-hackers` org, preferable a code owner from that repository.
 * If the author is not a member of the GitHub `flutter-hackers` org then they will need **at least 2 other reviews** from members of the `flutter-hackers` org.
 * If any reviewer requests changes then regardless of the number of approvals you have the pull request will not be merged by the Autosubmit bot and the `autosubmit` label will be removed until the reviewer who requested changes has approved the pull request.
 
 ### Check Runs and Statuses
-When a pull request is opened the Cocoon backend service will create a number of test runs that will be used to validate the code change that was submitted. Once the `autosubmit` label is added, the Autosubmit bot will check the statuses of those tests and will not merge the pull request until all of them have succeeded. If the any of the tests have failed the `autosubmit` label will be removed and the bot will no longer process that pull request. However the developer can re-trigger the test and add the label again to retry.
+
+When a pull request is opened the Cocoon backend service will create a number of test runs that will be used to validate the code change that was submitted. Once the `autosubmit` label is added, the Autosubmit bot will check the statuses of those tests and will not merge the pull request until all of them have succeeded. If any of the tests have failed the `autosubmit` label will be removed and the bot will no longer process that pull request. However the developer can re-trigger the test and add the label again to retry.
```

Comment:
Maybe this could use some more commas? Probably not a huge deal either way.

```suggestion
When a pull request is opened, the Cocoon backend service will create a number of test runs that will be used to validate the code change that was submitted. Once the `autosubmit` label is added, the Autosubmit bot will check the statuses of those tests and will not merge the pull request until all of them have succeeded. If any of the tests have failed, the `autosubmit` label will be removed and the bot will no longer process that pull request. However, the developer can re-trigger the test and add the label again to retry.
```

Reply:
This doc needs some other formatting and copy fixes, so I'll leave that to a separate PR by me or someone else. Thanks for calling this out though!